packages/core/src/tools/create-lambda-handler.js

```diff
@@ -30,17 +30,15 @@ const isRequestOrFunction = (obj) => {
 
 const extendAppRaw = (base, extension) => {
   const keysToOverride = [
```

Comment:
After looking over [the schema](https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md), I added `inputFields` as well as some other fields here since those seemed most likely for users to want to override with external actions. Please let me know if there are others that seem like a good fit.